### PR TITLE
Project Manifests should show Folders as members.

### LINKED ISSUE #422: Project Manifests should have IIIF Folders as members, not boxes

### LINKED ISSUE #423: Project Manifests should not display EphemeraTerms as members.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #423, closes #422.

### CODE REVIEW

File: app/queries/find_grandchildren.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  attr_reader :query_service
+  delegate :resource_factory, to: :query_service
+  delegate :orm_class, to: :resource_factory
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def find_grandchildren(resource:)
+    query_service.find_members(resource: resource).flat_map do |child|
```
Comment (escowles) [inline app/queries/find_grandchildren.rb]: With https://github.com/pulibrary/figgy/pull/397, the members could either be folders or boxes.  So we should probably filter the members to make sure we're only getting the members of boxes (and not getting FileSets too).  And then we'll want to get the directly-attached folders too.

Comment (tpendragon) [reply app/queries/find_grandchildren.rb]: Oh, yeah, okay.

Comment (hackmastera) [reply app/queries/find_grandchildren.rb]: sounds like `find_grandchildren` is no longer the correct method name.

Comment (escowles) [reply app/queries/find_grandchildren.rb]: `find_children_and_grandchildren_of_type` doesn't have the same ring to it

**Pull request merged**